roles/artemis/tasks/artemis_setup.yml

```diff
@@ -76,6 +76,26 @@
     group: "{{ artemis_user_group }}"
   when: artemis_data_export_path is not none and artemis_data_export_path != "" and artemis_computed_is_core_node
 
+- name: Create tmp directory
+  become: true
+  file:
+    path: "{{ artemis_tmp_directory }}"
+    state: directory
+    mode: "775"
+    owner: "{{ artemis_user_name }}"
+    group: "{{ artemis_user_group }}"
+  when: if artemis_tmp_directory is defined and artemis_tmp_directory is not none and artemis_tmp_directory
+
+- name: Turn the tmp directory into a tmpfs
+  become: true
+  ansible.posix.mount:
+    path: "{{ artemis_tmp_directory }}"
+    src: tmpfs
+    fstype: tmpfs
+    opts: defaults,noatime,nosuid,nodev,noexec,mode=1777,size=1G
+    state: mounted
+  when: if artemis_tmp_directory is defined and artemis_tmp_directory is not none and artemis_tmp_directory
```

Comment:
Might make sense to use consistent conditions (c.f. https://github.com/ls1intum/artemis-ansible-collection/pull/150/files#diff-69b61f905e1fea6a99aacb30650956d51d59a313e9b38c6cab7440348011f49fR94)